src/pytorch_lightning/callbacks/model_checkpoint.py

```diff
@@ -369,7 +369,8 @@ def _save_checkpoint(self, trainer: "pl.Trainer", filepath: str) -> None:
         # notify loggers
         if trainer.is_global_zero:
             for logger in trainer.loggers:
-                logger.after_save_checkpoint(proxy(self))
+                if hasattr(logger, "after_save_checkpoint"):
```

Comment:
@Borda where did you see this fail? The base logger interface includes this method.

Reply:
it is the initial log, or you can just run `tests/legacy/simple_classif_training.py` with just install PL without loggers :chicken:

Reply:
`PYTHONPATH=..:$PYTHONPATH python simple_classif_training.py` also doesn't fail with this change remove

Reply:
> The job in the description shows a different error.

which one? 
`AttributeError: 'CSVLogger' object has no attribute 'save_dir'`
and the I continued...

Reply:
but if this is fine for you I l remove it :)